Check for submissionsEnabled on Submit page

damianmarti opened the pull request:
It was showing the Submit form (the API endpoint is already checking if the submissions are enabled).

Pabl0cks reviewed (approved):
Nice catch Damu! ♥ 

Not sure why I implemented the `submissionsEnabled` check in the `SubmitButton` component, I think it's better your approach to not show the submission form, since it can have innacurate data from other hackathons🙏

Should we delete the `submissionsEnabled` logic from the button since it shouldn't be needed anymore?


Nitpick: pushed a styling change to have the same style of /admin "Access Denied" msg.

damianmarti commented:
> Nice catch Damu! ♥
> 
> Not sure why I implemented the `submissionsEnabled` check in the `SubmitButton` component, I think it's better your approach to not show the submission form, since it can have innacurate data from other hackathons🙏
> 
> Should we delete the `submissionsEnabled` logic from the button since it shouldn't be needed anymore?
> 
> Nitpick: pushed a styling change to have the same style of /admin "Access Denied" msg.

Removed the `submissionsEnabled` logic from the button!

Thanks!!

Pabl0cks commented:
> Removed the submissionsEnabled logic from the button!

Great! Merging 🙏
